concent_api/core/tests/utils.py

```diff
@@ -67,6 +68,14 @@ def add_time_offset_to_date(base_time: str, offset: int) -> str:
     )
 
 
+def generate_uuids(number_of_uuids: int) -> List[str]:
+    random.seed(123210912)
+    a = "%32x" % random.getrandbits(128)
+    rd = a[:12] + '4' + a[13:16] + 'a' + a[17:]
```

Comment:
Maybe it would be possible to choose some more meaningful names for those.

Reply:
done